Containerfile

```diff
@@ -495,3 +495,43 @@ RUN /tmp/image-info.sh && \
     mkdir -p /var/lib/bluetooth && \
     chmod -R 755 /var/lib/bluetooth && \
     ostree container commit
+
+FROM bazzite as bazzite-dx
+
+ARG IMAGE_NAME="${IMAGE_NAME}"
+ARG IMAGE_VENDOR="${IMAGE_VENDOR}"
+ARG IMAGE_FLAVOR="${IMAGE_FLAVOR}"
+ARG BASE_IMAGE_NAME="${BASE_IMAGE_NAME}"
+ARG FEDORA_MAJOR_VERSION="${FEDORA_MAJOR_VERSION}"
+
+COPY system_files/deck/shared system_files/deck/${BASE_IMAGE_NAME} /
```

Comment:
Instead of copying Deck files, we could move from this being an image target to an option so that DX desktop and DX Deck images can exist separately by adding another build flag that distinguishes itself between standard images and DX images